src/sentry/static/sentry/app/utils.tsx

```diff
@@ -61,21 +61,17 @@ function objectMatchesSubset(obj, other, deep) {
   return true;
 }
 
-// XXX(dcramer): the previous mechanism of using _.map here failed
-// miserably if a param was named 'length'
-export function objectToArray(obj) {
-  const result = [];
-  for (const key in obj) {
-    result.push([key, obj[key]]);
-  }
-  return result;
+export function objectToArray<T>(
+  obj: {[s: string]: T} | ArrayLike<T>
```

Comment:
When is it not a regular object?

Reply:
I'm just going to use `export const objectToArray = Object.entries;`